scripts/export-script.py

```diff
@@ -0,0 +1,94 @@
+import sys
```

Comment:
I think we should move this file to a different location. The `scripts` folder is for scripts that customers can run locally. 

Maybe create another scripts folder under /bulkExport

Reply:
Sure thing. Moving the code to the `bulkExport` folder.